applications/rubintv/templates/deployment-workers.yaml

```diff
@@ -0,0 +1,134 @@
+          valueFrom:
+            secretKeyRef:
+              name: {{ include "rubintv.fullname" . }}-secret
+              key: "redis-password"
+        - name: "ENVIRONMENT_URL"
+          value: {{ .Values.global.baseUrl }}
+        - name: "SAFIR_PATH_PREFIX"
+          value: {{ .Values.workers.pathPrefix | quote }}
+        - name: "RUN_ARG"
+          value: "{{ .Values.workers.script }} {{ .Values.siteTag }}"
+        {{- if (not .Values.workers.debug) }}
+        - name: "SAFIR_PROFILE"
+          value: "production"
+        {{- end }}
+        {{- with .Values.workers.env }}
+        {{- toYaml . | nindent 8 }}
+        {{- end }}
+        {{- if .Values.workers.envSecrets }}
+        {{- range $env := .Values.workers.envSecrets }}
+        - name: {{ $env.name }}
+          valueFrom:
+            secretKeyRef:
+              name: {{ $env.secretName }}
+              key: {{ $env.secretKey }}
+        {{- end }}
+        {{- end }}
+        image: "{{ .Values.workers.image.repository }}:{{ .Values.workers.image.tag | default .Chart.AppVersion }}"
+        imagePullPolicy: {{ .Values.workers.image.pullPolicy | quote }}
+        ports:
+          - name: "http"
+            containerPort: 8080
+            protocol: "TCP"
+        {{- with .Values.workers.resources }}
+        resources:
+          {{- toYaml . | nindent 10 }}
+        {{- end }}
+        securityContext:
+          allowPrivilegeEscalation: false
+          capabilities:
+            drop:
+              - ALL
+          readOnlyRootFilesystem: false
+          # Sigh, but what can you do?
+        volumeMounts:
+        - name: "tmp"
+          mountPath: "/tmp"
+        - name: "user-secrets"
+          mountPath: "/etc/secrets"
+        {{- range $vol := .Values.workers.volumes }}
+        - name: {{ $vol.name }}
+          mountPath: {{ $vol.mountPath }}
+        {{- end }}
+      initContainers:
+      - name: "secret-perm-fixer"
+        image: "busybox"
+        command:
+        - "/bin/sh"
+        - "-c"
+        - |
+          cp /secrets/* /etc/secrets && \
+          chown {{ $.Values.workers.uid }}:{{ $.Values.workers.gid }} \
+           /etc/secrets/* && \
+          chmod 0600 /etc/secrets/*
+        volumeMounts:
+        - name: "secrets"
+          mountPath: "/secrets"
+        - name: "user-secrets"
+          mountPath: "/etc/secrets"
+      volumes:
+      - name: "tmp"
+        emptyDir: {}
+      - name: "secrets"
+        secret:
+          secretName: {{ include "rubintv.fullname" . }}-secret
+      - name: "user-secrets"
+        emptyDir: {}
+      {{- range $vol := .Values.workers.volumes }}
+      - name: {{ $vol.name | quote }}
+        {{ with $vol.persistentVolumeClaim }}
+        persistentVolumeClaim:
+          claimName: {{ .name | quote }}
+        {{- end }}
+      {{- end }}
+      securityContext:
+        runAsNonRoot: true
+        runAsUser: {{ $.Values.workers.uid }}
+        runAsGroup: {{ $.Values.workers.gid }}
```

Comment:
Likewise here, shouldn't need the `$`.

Reply:
These were left over from adapting rapid-analysis, where all of this is inside a loop over startup scripts.